aws_lambda_builders/workflows/python_pip/packager.py

```diff
@@ -56,6 +55,14 @@ class PackageDownloadError(PackagerError):
     pass
 
 
+def get_lambda_abi(runtime):
+    return {
+        "python2.7": "cp27mu",
+        "python3.6": "cp36m",
+        "python3.7": "cp37m"
+    }.get(runtime)
```

Comment:
Is there any way we can ever get here with an unsupported `runtime`? If so do we want to raise something here like `UnsupportedRuntimeError` with a property for the `runtime` value? Or is this impossible, because of the validator? If its impossible I think I'd rather `[runtime]` so we at least get a very obvious `KeyError` instead of a `None` silently sneaking in.

Reply:
Ok yea. I think I like having an explicit error for it. Thoughts?